src/trove/chart.js.rkt

```diff
@@ -697,9 +1266,246 @@ a-series = from-list.grouped-bar-chart(
     '25 to 44 Years',
     '45 to 64 Years',
     '65 Years and Over'])
-render-chart(a-series).display()
-  }
-  @(in-image "grouped-bar-chart")
+    render-chart(grouped-series).display()} ,(in-image "grouped-bar-chart-example"))
+  `(@{stacked-series = from-list.stacked-bar-chart(
+  [list: 'CA', 'TX', 'NY', 'FL', 'IL', 'PA'],
+  [list:
+    [list: 2704659,4499890,2159981,3853788,10604510,8819342,4114496],
+    [list: 2027307,3277946,1420518,2454721,7017731,5656528,2472223],
+    [list: 1208495,2141490,1058031,1999120,5355235,5120254,2607672],
+    [list: 1140516,1938695,925060,1607297,4782119,4746856,3187797],
+    [list: 894368,1558919,725973,1311479,3596343,3239173,1575308],
+    [list: 737462,1345341,679201,1203944,3157759,3414001,1910571]],
+  [list:
+    'Under 5 Years',
+    '5 to 13 Years',
+    '14 to 17 Years',
+    '18 to 24 Years',
+    '25 to 44 Years',
+    '45 to 64 Years',
+    '65 Years and Over'])
+    render-chart(stacked-series).display()} ,(in-image "stacked-bar-chart-example"))
+  ]
+
+  @method-doc["DataSeries" "multi-bar-chart-series" "colors"]
+  @repl-examples[
+   `(@{render-chart(grouped-series.colors([list: red, orange, blue])).display()} ,(in-image "grouped-bar-chart-colors-example"))
+   `(@{render-chart(stacked-series.colors([list: red, orange, blue])).display()} ,(in-image "stacked-bar-chart-colors-example"))
+  ]
+
+  @method-doc["DataSeries" "multi-bar-chart-series" "sort"]
+  @repl-examples[
+   `(@{render-chart(grouped-series.sort()).display()} ,(in-image "grouped-bar-chart-sort-example"))
+   `(@{render-chart(stacked-series.sort()).display()} ,(in-image "stacked-bar-chart-sort-example"))
+  ]
+
+  @method-doc["DataSeries" "multi-bar-chart-series" "sort-by"]
+  @repl-examples[
+   `(@{descending-cmp = {(a, b): a > b}
+   eq = {(a, b): a == b}
+   render-chart(grouped-series.sort-by(descending-cmp, eq)).display()} ,(in-image "grouped-bar-chart-sort-by-example"))
+   `(@{descending-cmp = {(a, b): a > b}
+   eq = {(a, b): a == b}
+   render-chart(stacked-series.sort-by(descending-cmp, eq)).display()} ,(in-image "stacked-bar-chart-sort-by-example"))
+  ]
+
+  @method-doc["DataSeries" "multi-bar-chart-series" "sort-by-label"]
+  @repl-examples[
+   `(@{descending-str-cmp = {(a, b): a > b}
+   eq = {(a, b): a == b}
+   render-chart(grouped-series.sort-by-label(descendong-str-cmp, eq))
```

Comment:
`descending`?

In general, this means to me that these examples may not have been _run_ before pasting them into the documentation (because this is a clear unbound id). I think it would be helpful to, before we merge this, check that all of the examples in the docs run.

Reply:
The process I used was to write the code in code.pyret create the image, screenshot it, and then copy and paste the code into repl-example. I'm not sure how this error made it through but I'll make sure to do one last pass through all the examples!